src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC } from 'react';
+import { Person } from '../../types';
+import { PeopleTableHead } from '../PeopleTableHead/PeopleTableHead';
+import { PeopleTableBody } from '../PeopleTableBody/PeopleTableBody';
+
+interface Props {
```

Comment:
Similar names are okay for a small project, but on large projects we will have many interfaces and types, so it is better to give more detailed names, for example `PeopleTableProps`.
This applies to all code.